lib/topology/logging.py

```diff
@@ -131,6 +131,27 @@ def log_dir(self, log_dir):
 
 
 @add_metaclass(ABCMeta)
+class StdOutLogger(BaseLogger):
+    """
+    Logger that logs to the standard output.
+    """
+
+    def __init__(
+        self, nameparts, propagate=False, level=logging.NOTSET, log_dir=None,
+        *args, **kwargs
+    ):
+        super(StdOutLogger, self).__init__(
+            nameparts, propagate=propagate, level=level, log_dir=log_dir,
+            *args, **kwargs
+        )
+
+        self.logger.addHandler(logging.StreamHandler())
+
+    def log(self, message):
+        self.logger.info(message)
```

Comment:
Ok, I'm submitting changes now, I just dont't understand what you mean with "list log type".